4-WebApp-your-API/4-1-MyOrg/Client/Services/TodoListService.cs

```diff
@@ -71,8 +71,6 @@ public async Task DeleteAsync(int id)
             {
                 return;
             }
-
-            throw new HttpRequestException($"Invalid status code in the HttpResponseMessage: {response.StatusCode}.");
```

Comment:
>HttpRequestException [](http://example.com/codeflow?start=22&length=20)

so if the delete failed, the client app/user gets no idea that it happened??

Reply:
you're looking into old code